jwst/stpipe/core.py

```diff
@@ -19,13 +19,21 @@
 
 class JwstStep(Step):
 
+    datamodel_access_times = 0
```

Comment:
This attribute is shared by all instances of `JwstStep` which will lead to unintended consequences.

Reply:
ok, I'll remove it